components/Shimmer/Shimmer.tsx

```diff
@@ -1,18 +1,25 @@
 import { cn } from '@/utils/utils'
 
 type ShimmerProps = {
+    /** Tailwind classes to override default styling */
     className?: string;
 }
 
 export const Shimmer = ({
     className,
 }: ShimmerProps) => {
     return <div className={cn("animate-pulse relative overflow-hidden rounded-lg mx-3 bg-indigo-50 opacity-90 max-w-full h-full", className)} data-testid="shimmer" />
-
 };
 
+type ListShimmerProps = {
+    /** Tailwind classes to container's override default styling */
+    className?: string;
+
+    /** Number of the shimmer inside the container   */
+    count: number
+}
 
-export const ListShimmer = ({ className, count }: { className?: string, count: number }) => {
+export const ListShimmer = ({ className, count }: ListShimmerProps) => {
```

Comment:
- let's not export multiple components in base components, only export the shimmer component and create a new file for `ListShimmer`.